Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -1,37 +1,422 @@
-﻿namespace BootCamp.Chapter
+﻿using System.Globalization;
+using System.Text;
+
+namespace BootCamp.Chapter
 {
     public static class BalanceStats
     {
+        private const string currencySymbol = "¤";
+        private const string invalidMessage = "N/A.";
+        private const string messageEnd = ".";
+        private static readonly NumberFormatInfo numberFormatInfo = new NumberFormatInfo() { NumberDecimalSeparator = messageEnd };
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove these redundant parentheses.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4970157)

Reply:
Fixed in [2db6c28](https://github.com/csinn/CSharp-From-Zero-To-Hero/pull/161/commits/2db6c282219933cc661146c542a210b0f73f44cf)